gsa/src/web/pages/results/row.js

```diff
@@ -46,7 +46,7 @@ import TableData from '../../components/table/data.js';
 import ResultDelta from './delta.js';
 
 const Hostname = glamorous.span({
-  fontSize: '10px',
+  fontSize: '0.8em',
```

Comment:
Instead of decreasing font size, you could use gmp/utils/string.js's shorten() to cut off names that are too long.